react-app-drawer-component/my-app/src/App.tsx

```diff
@@ -0,0 +1,17 @@
+import './App.css';
+import { AppDrawer } from './AppDrawer';
+
+const menuItems = [
+  'The Legend of Zelda',
+  'A Link to the Past',
+  'Ocarina of Time',
+  'The Wind Waker',
+  'Breath of the Wild',
+  'Tears of the Kingdom',
+];
+
+function App() {
+  return <AppDrawer header={menuItems} />;
```

Comment:
As discussed, this is where you should draw the main content, including the selected menu item.